Netro Sprinklers.indigoPlugin/Contents/Server Plugin/plugin.py

```
@@ -270,10 +263,59 @@ def _make_api_call(self, url, request_method="get", data=None):
+                                    f"consider increasing polling interval in plugin preferences"
+                                )
+                            except (ValueError, TypeError):
+                                # Fallback to fixed delay if we can't parse reset time
+                                self.throttle_next_call = datetime.now() + timedelta(minutes=THROTTLE_LIMIT_TIMER)
+                                error_msg = (
+                                    f"netro api rate limit exceeded, "
+                                    f"will retry in {THROTTLE_LIMIT_TIMER} minutes, "
+                                    f"consider increasing polling interval"
+                                )
+
+                            self.logger.error(error_msg)
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)